src/tests/App.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```

Comment:
This test file looking pretty good.  It's nice that we're testing several different things about the app here, and that they're arranged in the order of complexity of the tasks (button EXISTENCE first, testing to see if it renders events after).  

It would be nice to see some more tests here for:
1) failure states, proper error handling, etc.  What do we expect the app to do when things go wrong with an API call, for instance.  We could mock a failed API call and make sure that the app follows the proscribed behaviour (i.e. popping up a message or displaying it in a div with class "error-display" or whatever you design your app to do in the case of each api call and each type of error it might run into).
2) even more granular tests of the app's functionality representing further steps of development past the displaying of events on the page.  We've gotten up to the rendering of events, but we could be looking at each event or even passing in a mocked list of events that have pre-set content that we can verify is displayed in the correct location, format, etc.  We could look at the interact-ability of an event, or even (in combination with point #1 above) we could examine how events display when passed an array of events that will cause SOME sort of error.